modules/aidemBidAdapter.js

```diff
@@ -406,10 +412,26 @@ function hasValidVideoParameters(bidRequest) {
   return valid
 }
 
+function passesRateLimit(bidRequest) {
+  const rateLimit = deepAccess(bidRequest, 'params.rateLimit', 1);
+  if (!isNumber(rateLimit) || rateLimit > 1 || rateLimit < 0) {
+    logError('AIDEM Bid Adapter: invalid rateLimit (must be a number between 0 and 1)', { bidder: BIDDER_CODE, code: ERROR_CODES.INVALID_RATELIMIT });
+    return false
+  }
+  if (rateLimit !== 1) {
+    const randomRateValue = Math.random()
```

Comment:
Why is this a random generated value?

Reply:
Hello Chris! We have a rate limiting feature, used to limit the QPS sent to AIDEM SSP. In case this option is enabled by the publisher (rateLimit !== 1), we generate a random value to decide if we will send or not the bidRequest to our SSP. The Math.random() call happens only if the publisher has enabled this feature.